blazor/mention/code-snippet/filter-type.razor

```diff
@@ -0,0 +1,42 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Data
+
+<SfMention TItem="EmployeeData" Query="@Query" PopupWidth="250px" suggestionCount=6 FilterType="Syncfusion.Blazor.DropDowns.FilterType.StartsWith">
+    <TargetComponent>
+        <div id="commentsMention" placeHolder="Type @@ and tag employee"></div>
+    </TargetComponent>
+    <ChildContent>
+        <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
+        <MentionFieldSettings Text="FirstName" Value="EmployeeID"></MentionFieldSettings>
+    </ChildContent>
+</SfMention>
+
+<style>
+
+    #commentsMention {
+        min-height: 100px;
+        border: 1px solid #D7D7D7;
+        width: 600px;
+        font-size: 14px;
+        padding: 8px;
+        border-radius: 4px;
+    }
+
+    div#commentsMention[placeholder]:empty:before {
+        content: attr(placeholder);
+        color: #555;
+    }
+
+</style>
+
+@code {
+
+    public class EmployeeData
+    {
+        public string FirstName { get; set; }
+        public string Country { get; set; }
```

Comment:
Add the EmployeeID , Check all code snippets, You should give the Value mapping field. Because we have passed the value on a select event.

Reply:
I have mapped the County to value field sir.